CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # Change Log
 
-# Latest updates
+## Latest updates
 
-From v1.6.6, You can check all of our changes from [Releae Page](https://github.com/Ikuyadeu/vscode-R/releases)
+From v1.6.6, You can check all of our changes from [Release Page](https://github.com/Ikuyadeu/vscode-R/releases)
 
-# [1.6.6](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.6.6)
+## [1.6.6](https://github.com/Ikuyadeu/vscode-R/releases/tag/v1.6.6)
```

Comment:
Does this mean we don't write updates from collaborators in this changelog any more? It looks a bit un-informative as a quick preview of what has been updated in this release. A typical practice is that the changelog lists the most important updates and the release page shows the updates in details.

Reply:
I see, actually, this is unhelpful.
So I'll add important update information as the `Highlight` it does not include the engineering update such as lint fix and refactoring like this PR.

Reply:
Yes exactly, I guess most users only care about new features and some important bug fixes and changes of behavior.

Reply:
In the future, we may demand the author of a PR to update the changelog if we think the PR is important to mention and then we could review the changelog updates too within the PR.

Reply:
I agree with you.
Official vscode makes the PR changelog automatically. We may imitate that approach.